riotOS-nrf52840dk-ble/wolfboot.mk

```diff
@@ -0,0 +1,71 @@
+ifdef WOLFBOOT_OFFSET
```

Comment:
is the WOLFBOOT_OFFSET setting relevant?

Reply:
Yes, it's used by the wolfboot.mk to decide whether to change the offset of the linked application. When it's not defined, the application is linked to run from the entry point (no bootloader)

Reply:
Thanks for the explanation.